modular_skyrat/modules/importedvendors/code/vendor_food.dm

```diff
@@ -0,0 +1,276 @@
+	. = ..()
+	if(hot_and_steamy)
+		overlays += mutable_appearance('icons/effects/steam.dmi', "steam_triple", ABOVE_OBJ_LAYER)
+
+/obj/item/food/vendor_tray_meal/examine_more(mob/user)
+	. = ..()
+	. += span_notice("<i>You browse the back of the box...</i>")
+	. += "\t[span_warning("Warning: Packaged in a factory where every allergen known is present.")]"
+	. += "\t[span_warning("Warning: Contents might be hot.")]"
+	. += "\t[span_info("Per 200g serving contains: 8g Sodium; 25g Fat, of which 22g are saturated; 2g Sugar.")]"
+	return .
+
+/obj/item/food/vendor_tray_meal/burger
+	name = "\improper NT-Meal: Cheeseburger"
+	desc = "A pretty sad looking burger with a kinda soggy bottom bun and highlighter yellow cheese."
+	icon_state = "foodtray_burg"
+	tastes = list("bread" = 2, "meat?" = 2, "cheese?" = 2, "laziness" = 1)
+	foodtypes = MEAT | GRAIN | DAIRY
+
+/obj/item/food/vendor_tray_meal/chicken_sandwich
+	name = "\improper NT-Meal: Spicy Chicken Sandwich"
+	desc = "A pretty sad looking chicken sandwich, the 'meat' itself covered in so many manufactured spices that it has become an eerie red color."
+	icon_state = "foodtray_chickiesandwich"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/capsaicin = 10)
+	tastes = list("bread" = 2, "chicken?" = 2, "overwhelming spice" = 2, "laziness" = 1)
+	foodtypes = MEAT | GRAIN | DAIRY
+
+/obj/item/food/vendor_tray_meal/ramen
+	name = "\improper メインコース: Homestyle Noodles"
+	desc = "A brick of the finest factory made ramen, with a small amount of rehydrated vegetables and herbs floating around."
+	icon_state = "foodtray_noodle"
+	tastes = list("cheap noodles" = 2, "laziness" = 1)
+	foodtypes = GRAIN | VEGETABLES
+
+/obj/item/food/vendor_tray_meal/sushi
+	name = "\improper メインコース: Fresh Carp Rolls"
+	desc = "A pair of sushi rolls, the appearance of which would suggest that the label is lying to you."
+	icon_state = "foodtray_gasstationsushi"
+	tastes = list("imitation space carp" = 2, "stale rice" = 2, "laziness" = 1)
+	foodtypes = GRAIN | SEAFOOD
+
+/obj/item/food/vendor_tray_meal/beef_rice
+	name = "\improper メインコース: Beef and Fried Rice"
+	desc = "A few slices of seemingly grilled beef, paired with far too much rice for how much meat there is."
+	icon_state = "foodtray_beefnrice"
+	tastes = list("cheap beef" = 1, "rice" = 3, "laziness" = 1)
+	foodtypes = GRAIN | MEAT
+
+/obj/item/food/vendor_tray_meal/pesto_pizza
+	name = "\improper Main Course - Type M: Pesto Pizza"
+	desc = "A rectangular pizza with a suspiciously bright green pesto in place of the standard tomato sauce."
+	icon_state = "foodtray_pestopizza"
+	tastes = list("tomato?" = 2, "cheese?" = 2, "herbs" = 1, "laziness" = 1)
+	foodtypes = GRAIN | DAIRY | VEGETABLES
+
+/obj/item/food/vendor_tray_meal/baked_rice
+	name = "\improper Main Course - Type M: Baked Rice and Grilled Cheese"
+	desc = "Some sub-par looking fleet style rice, with a generous hunk of quite literally grilled cheese."
```

Comment:
```suggestion
	desc = "Some sub-par looking fleet style rice, with a quite literally grilled chunk of cheese."
```
Helps distinguish this as different from the sandwich (Alternatively, you can put the kind of cheese, like "Grilled Halloumi Cheese" but that would require you to know the kind of cheese)